Cargo.toml

```diff
@@ -9,7 +9,10 @@ members = [
     "crates/pl011-driver",
     "crates/uart-driver",
     "crates/uart-interface-types",
-    "crates/ethernet-interface-types",
     "crates/eth-driver",
     "crates/eth-client",
 ]
+
+[workspace.dependencies]
+sel4cp = { git = "https://github.com/coliasgroup/rust-seL4", rev = "0997c8d54426d590a59b9894bfac9d47d804188e", default-features = false }
+sel4-shared-ring-buffer = { git = "https://github.com/protoben/rust-seL4", rev = "58ae0cfe5cf7a5c8dc13f315062ec94dee784c47", default-features = false }
```

Comment:
please use the upstream version, Nick merged your fix

Reply:
See the comment above. There appear to be interface changes, so I'll need to figure out the new one.